docs/tutorial/inputs/README.md

```diff
@@ -0,0 +1,68 @@
+---
+order: 2
+---
+
+# Inputs
+
+Flows can have Input. They are parameters provided when starting a flow. They help make your flow more dynamic and reusable.
+
+## Defining inputs
+
+We defined Inputs in the `inputs` section of the flow file. They must have a `name` and a `type`.
+You can also set a `default` value.
+
+```yaml
+inputs:
+  - name: isTutorial
+    type: BOOLEAN
+    default: true
+```
+
+| Type    | Description                                                            |
+|---------|------------------------------------------------------------------------|
+| STRING  | No control is done on this input type (no parsing), can be any string. |
+| INT     | Must be a valid integer (without any decimals).                        |
+| BOOLEAN | Must be a valid true or false as string.                               |
```

Comment:
Can you replace with something like "Kestra support multiple input types STRING, INT, BOOLEAN, FILE, ..."
Avoid table like this as a tutorial is better if you wan read it fluently.